apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,68 @@
-import { Component, OnInit } from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Signal,
+  ViewEncapsulation,
+  effect,
+  inject,
+} from '@angular/core';
+import { CardComponent } from '../../ui/card/card.component';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CityStore } from '../../data-access/city.store';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card [list]="cities()" customClass="bg-light-green">
+      <ng-template let-item="item" #listItem>
+        {{ item.name }}
+        <ng-container ngProjectAs="delete-button">
+          <button (click)="delete(item.id)">
+            <img class="h-5" src="assets/svg/trash.svg" />
+          </button>
+        </ng-container>
+      </ng-template>
+
+      <ng-container ngProjectAs="add-button">
+        <button
+          class="border border-blue-500 bg-blue-300 p-2 rounded-sm"
+          (click)="add()">
+          Add
+        </button>
+      </ng-container>
+    </app-card>
+  `,
   standalone: true,
-  imports: [],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [CardComponent],
 })
-export class CityCardComponent implements OnInit {
-  constructor() {}
+export class CityCardComponent {
+  private store = inject(CityStore);
+  private fetchCities = toSignal(inject(FakeHttpService).fetchCities$, {
+    initialValue: [],
+  });
```

Comment:
http call are still async, so don't use signal, make thing more complicated. We will still need rxjs. We should not transform everything, just what needs to be bind to the template.

Reply:
Yes I will modify this